android/src/main/java/com/tailscale/ipn/ui/viewModel/ExitNodePickerViewModel.kt

```diff
@@ -51,15 +52,28 @@ class ExitNodePickerViewModel(private val nav: ExitNodePickerNav) : IpnViewModel
       MutableStateFlow(TreeMap())
   val mullvadBestAvailableByCountry: StateFlow<Map<String, ExitNode>> = MutableStateFlow(TreeMap())
   val anyActive: StateFlow<Boolean> = MutableStateFlow(false)
-    val isRunningExitNode: StateFlow<Boolean> = MutableStateFlow(false)
+  val isRunningExitNode: StateFlow<Boolean> = MutableStateFlow(false)
+
+  val allowLANAccessSetting =
+      Setting(
+          R.string.allow_lan_access,
+          SettingType.SWITCH,
+          isOn = MutableStateFlow(Notifier.prefs.value?.ExitNodeAllowLANAccess),
+          onToggle = {
+            LoadingIndicator.start()
+            toggleAllowLANAccess {
+              LoadingIndicator.stop()
+              // (jonathan) TODO: Error handling
```

Comment:
We can probably swallow the errors for all of these.  There's nothing the user can do, and LocalAPI is not going to fail in the ways it can on iOS.